opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/perl/PerlLexer.java

```diff
@@ -682,4 +671,58 @@ public HereDocSettings(String terminator, int state) {
 
         public int getState() { return state; }
     }
+
+    class PerlLexerData {
```

Comment:
can it be static?

Reply:
Excised Perl stacking, which was accidentally ported from Ruby